README.md

```diff
@@ -349,7 +349,7 @@ tj.see().then(function(objects) {
 Sample resposne:
```

Comment:
just saw a typo in 'resposne' (this is my fault).. would you mind fixing it?  :D